src/Npgsql.NodaTime/Npgsql.NodaTime.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <VersionPrefix>1.0.0</VersionPrefix>
+    <VersionPrefix>4.0.4</VersionPrefix>
```

Comment:
Maybe 4.0.0? I don't see a reason to align patch versions with Npgsql - as Npgsql patch versions are released, I don't see a reason to make useless corresponding releases in all plugins. As long as major/minor are aligned that should be enough no?

Reply:
Going forward, I think letting the patches increment separately is a good decision. 

For this one, I'll keep 4.0.4, as I think it will be more obvious what's going on to existing consumers to see a jump from 1.0.0 to a fully matched version.